Antonio_Ramos_Documentation

```diff
@@ -381,3 +381,80 @@ Tab2 = Tab2.astype(convert_dict)
 
 - Problems left to resolve - 1. no cluster file is being generated even after 499 simulations 2. the find and replace done on the excel file needs to be automated in python 3. the code that produces the labs data file is not producing files which project correctly 
 
+8/8/2020
+
+- installying arcpy to python envinronment "programTrial" using following command in anaconda command prompt "conda install -c esri arcpy" 
+
+- error arcpy cannot run on python 3.7 so I will create a new environment that uses python 3.6 by running the following command in anaconda terminal prompt "conda create --name programTrial3.6 python=3.6" 
+
+- will install all packages used in programTrial and arcpy 
+
+8/9/2020
+- during creating of programTrial 3.6 multiple conflicts arose from the installation of arcpy 
+
+- will begin determining which order to install packages so that conflicts do not arise
+
+8/11/2020
+- removed the programTrial3.6 environment and created new environment where I install packages and programs in the following order 
+  1. Spyder 4.0.1. (Installed through the anaconda navigator) 
+  2. arcpy (anaconda command prompt command "conda install -c esri arcpy")
+  3. plotnine (anaconda command prompt "conda install -c conda-forge plotnine") 
+  4. geopandas (anaconda command prmopt "conda install-c conda-forge geopandas") 
+- all moduels and packed installed successfully
```

Comment:
Are there some typos here? It isn't clear what this means.

Reply:
This section is just the order in which we have to install packages in the python environment to be able to install arcpy. If you do not go in this order when installing packages in a new python environment, then you will have conflicting dependencies between the packages.

Reply:
Looking at this again, I'm thinking that "packed" should be "packages"?